docs/example/README.md

```diff
@@ -214,6 +227,48 @@ Wait for the changes to propagate and observe the counters increment for booksto
 
 - http://localhost:8082 - **bookstore-v2**
 
+## Permissive Traffic Policy Mode
+
+In permissive mode, application connectivity within the mesh is automatically configured by `osm-controller`. It can be enabled in the following ways.
+
+1. During install using `osm` cli:
+```bash
+osm install --enable-permissive-traffic-policy
+```
+
+2. Post install by updating the `osm-config` ConfigMap in the control plane's namespace (`osm-system` by default)
+```bash
+# Replace osm-system with osm-controller's namespace if using a non default namespace
+kubectl patch ConfigMap osm-config -n osm-system -p '{"data":{"permissive_traffic_policy_mode":"true"}}' --type=merge
+```
+
+### Deploy a service with permissive mode enabled
+
+Enable permissive mode as described above and verify that the counters on the `Bookbuyer` window are not incrementing anymore.
```

Comment:
When the permissive mode is enabled, I would expect the counter to increment. 
> The `Bookbuyer` application is able to purchase books from the `bookstore` service because permissive mode is enabled, thereby allowing connectivity between applications without the need for SMI traffic policies.

Reply:
I see why this can be confusing, removed it.